### ISSUE TITLE
FEATURE SUGGESTION- Lighter/Brighter Theme Option

### ISSUE THREAD

**Issue by MrFox14** [EchoDevG/trigger-fixer#7]
The current red and black theme is a bit scary to SOME, can you add an option that will make the theme look lighter/brighter colors? (Just add a button for it, it's up to you if you want the red and black theme default)

**Comment by echogarratt** [comment]
A button which changed the style sheet is all we'd need right @Mistyttm?

@MrFox14 what colour scheme do you think would work?

**Comment by EchoDevG** [comment]
@Mistyttm @MrFox14 We are migrating to GitLab in the hopes of easier builds and deploys

**Comment by Mistyttm** [comment]
> A button which changed the style sheet is all we'd need right @Mistyttm?
> 
> 
> 
> @MrFox14 what colour scheme do you think would work?

Yep, I'm currently working on creating the dark/light mode in the Firefox version, you can see my progress in the main branch, which is the most up to date and unstable version of the extension
